packages/loader/container-loader/src/test/deltaManager.spec.ts

```diff
@@ -176,6 +176,7 @@ describe("Loader", () => {
                     await tickClock(1000 * 1000);
                     assert.equal(counter, 0, "No messages sent after 99 ops");
                     tracker.scheduleSequenceNumberUpdate(generateOp(), false);
+                    await tickClock(1000 * 1000);
```

Comment:
I do not think that long time is required - I think just yielding is enough (there is some helper function for that somewhere). Or tickClock(1) or even zero should be sufficient.